crates/viewer/re_component_ui/src/recording_uri.rs

```diff
@@ -26,7 +26,10 @@ pub fn singleline_view_recording_uri(
             })
             .inner;
 
-        if response.clicked() {
+        if response.clicked() || response.middle_clicked() {
```

Comment:
I see the parent scope is surrounded by `#[cfg(not(target_arch = "wasm32"))]` - why?

Reply:
That's a good question, I noticed that too. I think it's unwarranted, but it's currently difficult to test. My next todo is to add the ability to add a new redap server from the UI. When we have that, we can test how stuff behaves with the web viewer and deal with this conditional.